app/src/main/java/com/kickstarter/ui/activities/ProjectActivity.kt

```diff
@@ -345,6 +359,14 @@ class ProjectActivity : BaseActivity<ProjectViewModel.ViewModel>(), CancelPledge
         }
     }
 
+    private fun setProjectActionCTA(stringRes: Int) {
+        project_action_button.setText(stringRes)
+        project_action_button.contentDescription = when (stringRes) {
+            R.string.Manage -> getString(R.string.Manage_your_pledge)
```

Comment:
Ah interesting, is this string mapping because the `contentDescription` is what is used by screen readers?

Reply:
yep! and normally for `Button`s it just reads the text